src/components/treeIconLegend/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Col, Image, Row, Typography } from 'antd';
+import {
+  ADOPTED_TREE_ICON_DESCRIPTION,
+  TREE_ICON_DESCRIPTION,
+  YOUNG_TREE_ICON_DESCRIPTION,
+} from '../../assets/content';
+import adoptedTreeIcon from '../../assets/images/adoptedTreeIcon.png';
+import treeIcon from '../../assets/images/treeIcon.png';
+import youngTreeIcon from '../../assets/images/youngTreeIcon.png';
+
+const { Paragraph, Text } = Typography;
+
+const StyledImage = styled(Image)`
+  display: inline-block;
+`;
+
+const CenterCol = styled(Col)`
+  text-align: center;
+`;
+
+const StyledParagraph = styled(Paragraph)`
+  line-height: 15px;
+  font-size: 0.9em;
```

Comment:
It worked well with the different browser sizes as opposed to passing a prop and setting some `px` value accordingly. I'm not too familiar with the pros and cons of using `em`, but this can be changed if needed. I just thought it was simpler.